src/utils/feature-flags.ts

```diff
@@ -23,6 +23,7 @@ export const getFeatureFlagsFromSiteInfo = (siteInfo: {
   ...(siteInfo.feature_flags || {}),
   // see https://github.com/netlify/pod-dev-foundations/issues/581#issuecomment-1731022753
   zisi_golang_use_al2: isFeatureFlagEnabled('cli_golang_use_al2', siteInfo),
+  netlify_build_frameworks_api: true,
```

Comment:
🤔 Were you initially thinking of feature flagging this and then you changed your mind and forgot to remove this? I might be missing something otherwise.

Reply:
No, the idea here is to always enable the Frameworks API in Netlify Build, which is still behind a feature flag in our build system.